src/main/java/se/kth/sda6/nysure/claim/Claim.java

```diff
@@ -0,0 +1,47 @@
+package se.kth.sda6.nysure.claim;
+
+import se.kth.sda6.nysure.user.User;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "claim")
+public class Claim {
+    @Id
+    @Column(name = "claimId")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long claimId;
+
+    @Column(name = "incidentDescription")
+    private String inDescr;
+
+    @ManyToOne()
+    private User user;
+
```

Comment:
In our claim, will we not need:
- photos linked to the policy
        -> before (null if they don't exist)
        -> after incident (null if they exist)
- policy number or other insurance specific data?

Reply:
we will not need? not sure what you meant, but I removed the user it will be in the insurance, and claim will instead be linked to insurance